ansible/roles/machine/provision/tasks/main.yml

```diff
@@ -35,12 +35,7 @@
         - python*-ipatests
 
 - name: get all packages
-  shell: rpm -qa
-  register: packages
-
-- name: display installed packages
-  debug:
-    var: packages.stdout
+  shell: rpm -qa > /vagrant/installed_packages.log
```

Comment:
Previously, you could check the installed packages on all machines. They should be the same on all machines -- but we're losing the option to verify and check, because this overwrites the file for each VM. With the current setup, that's ok, since all packages are the same on all machines (same template, same playbook executed). If you're find with this behavior, ack.